lib/sht4x/measurement.ex

```diff
@@ -23,6 +24,27 @@ defmodule SHT4X.Measurement do
   this is real temperature and humidity detected.
   """
   @spec from_raw(<<_::48>>) :: t()
+  def from_raw(<<raw_t::16, _crc1, raw_rh::16, _crc2>>)
+      when raw_t == 0x8000 and raw_rh == 0x8000 do
```

Comment:
```suggestion
      when raw_t == 0x8000 and raw_rh == 0x8000 do
```
The report also had `raw_t == 0x8001` sometimes. 

Looking at the conversion formulas, it seems like more values could indicate bad measurements as well. For example, `humidity_rh < 0 or > 100` seem bad too. I'm not sure if we know enough to handle that well in this PR since compensation algorithms might allow some weird numbers. However, `0x0000` and `0xffff` seem extreme enough that they may warrant similar treatment. Maybe raw temp validation gets refactored into a small set of functions rather than doing it in a guard?